docs/src/design/stack/field_ops.md

```diff
@@ -178,19 +178,25 @@ The effect on the rest of the stack is:
 * **No change** starting from position $1$.
 
 ## EXPACC
-The `EXPACC` operation pops top $4$ elements from the top of the stack, performs a single round of exponent aggregation, and pushes the resulting $4$ values onto the stack. The diagram below illustrates this graphically.
+The `EXPACC` operation computes one round of the expression $base^{exp}$. It is expected that `Expacc` is called at least `num_exp_bits` times, where `num_exp_bits` is the number of bits required to represent `exp`.
+
+It pops top $4$ elements from the top of the stack, performs a single round of exponent aggregation, and pushes the resulting $4$ values onto the stack. The diagram below illustrates this graphically.
 
 ![expacc](../../assets/design/stack/field_operations/EXPACC.png)
 
+Expacc is based on the observation that the exponentiation of a number can be computed by repeatedly squaring the base and multiplying the accumulator by the base when the least significant bit of the exponent is 1.
+
+For example, take $b^5 = (b^2)^2 \cdot b$. Over the course of 3 iterations ($5$ is $101$ in binary), the algorithm will compute $b$, $b^2$ and $b^4$ (placed in `base`). Hence, we want to multiply `base` in `acc` when $base = b$ and when $base = b^4$, which occurs on the first and third iterations (corresponding to the $1$ bits in the binary representation of 5).
```

Comment:
nit: same as the previous remark with respect to `base`

Reply:
fixed as above (and in the implementation of `op_expacc`)